Update worker-variables.mdx (#23948)
cloudflare/cloudflare-docs: src/content/docs/zaraz/variables/worker-variables.mdx

Once your Worker is published, configuring a Worker Variable is easy.

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/).
2. Go to **Tag Management** > **Tag Setup**
3. Click on the domain for which you want to configure variables. 

> 4. Navigate to **Variables** tab.

Reviewer: ```suggestion
4. Select the **Variables** tab.
```

> 5. Click **Create variable**.

Reviewer: ```suggestion
5. Select **Create variable**.
```

6. Save your variable.

## Using your Worker Variable

Now that your Worker Variable is configured, you can use it in your actions and triggers.

To use your Worker Variable:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/).
2. Go to **Tag Management** > **Tag Setup**
3. Click on the domain for which you want to configure variables. 

> 4. Click **Third-party tools**.

Reviewer 2: This is not necessary since Third-party tools tab is the default one

> 4. Click **Edit** next to a tool that you have already configured.

Reviewer: ```suggestion
4. Select **Edit** next to a tool that you have already configured.
```

5. Select an action or add a new one.

> 6. Click on the plus sign at the right of the text fields.

Reviewer: ```suggestion
6. Select the plus sign at the right of the text fields.
```
